airgap/client.go

```diff
@@ -94,17 +95,21 @@ func (c *clientImpl) GenerateProofOfPossessionSignature(privateKey *ecdsa.Privat
 	return signature, err
 }
 
-var abiParsers = []func() (*abi.ABI, error){
-	contracts.ParseReleaseGoldABI,
-	contracts.ParseAccountsABI,
-	contracts.ParseLockedGoldABI,
-	contracts.ParseElectionABI,
+var abiParsers = map[string]func() (*abi.ABI, error){
+	ReleaseGold:                            contracts.ParseReleaseGoldABI,
```

Comment:
qq why not do `registry.ReleaseGoldContractID.String()`?

Reply:
ReleaseGold doesn't exist in the registry because there is a deployment per beneficiary

Reply:
ah gotcha, no one canonical contract, so you have to hardcode the key